src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -117,6 +138,15 @@ Whenever possible, please avoid using ad hoc wording.
 
 Specific iEEG fields MUST be present:
 
+{{ MACROS___make_metadata_table(
+   {
+      "iEEGReference": "REQUIRED",
+      "SamplingFrequency": "REQUIRED",
```

Comment:
Here we should add something like: "The sampling frequency of data channels that deviate from the main sampling frequency SHOULD be specified in the `channels.tsv`file"

Reply:
```suggestion
      "SamplingFrequency": ("REQUIRED", "The sampling frequency of data channels that deviate from the main sampling frequency SHOULD be specified in the `channels.tsv` file."),
```